Frontend/src/Components/Dashboard/Dashboard.js

```diff
@@ -9,6 +9,7 @@ import {
 import { useEffect, useReducer, useState } from "react";
 import FaultsView from "../Faults/FaultsView";
 import DriverComms from "../GeneralData/DriverComms";
+import Communication from "../GeneralData/Communication";

```

Comment:
```suggestion
import Communication from "../Communication/Communication";
```
Sorry, this is on me for making a sucky project structure initially. Could you please move the Communication component and its assets to a new `Communication` directory?